INSTALL.md

```diff
@@ -0,0 +1,100 @@
+# Building and Installation
+
+## Dependencies
+
+Ensure the development version of the dependencies are installed. These are as follows:
+
+* Tango Controls 9.2.5a.
```

Comment:
I would not restrict it to this specific version of Tango. It should be compatible with the next Tango 9 versions too.
I think having Tango 9 is enough but we could simply specify that this has been tested successfully with tango 9.2.5a or a similar sentence...

Reply:
Yes, this was more of an issue with the tango.h changes I backed out, and should be fine now. This means the packages will change also. Will make this update across all the pull requests.